datasets/gov/kaunas/pagalve.csv

```diff
@@ -0,0 +1,10 @@
+id,dataset,resource,base,model,property,type,ref,source,prepare,level,access,uri,title,description
+,datasets/gov/kaunas/pagalve,,,,,,,,,,,,Kauno miesto savivaldybės teikiami duomenys,Kauno mieste surenkamas pagalvės mokestis
+,,,,,,,,,,,,,,
+,,,,gauta,,,id,,,,,,Gauta suma per periodą,
+,,,,,id,integer,,,,,open,,ID,
+,,,,,dt_nuo,date,,,,,open,,Pradžios data,periodo pradžios data
+,,,,,dt_iki,date,,,,,open,,Pabaigos data,periodo pabaigos data
+,,,,,suma,number,,,,,open,,suma,suma eurais
+,,,,,subjektu_kiekis,integer,,,,,open,,asmenų kiekis,paslaugą teikiančių asmenų kiekis
+,,,,,objektu_kiekis,integer,,,,,open,,objektų kiekis,paslaugos teikimo vietų (objektų) kiekis
```

Comment:
Kaip suprantu, duomenys yra agreguoti, gal yra galimybė teikti neagreguotus, nuasmenintus duomenis?

Reply:
Perduodant neagreguotus duomenis gali būti atskleista komercinė paslaptis, kadangi paslaugų tiekėjai žino kas kiek kokių patalpų turi ir atrasti koks pas ką buvo užimtumas.

Reply:
Kiti pakeitimai tinka

Reply:
Agregavimas nėra tas pats, kas nuasmeninimas arba konfidencialumo užtikrinimas. Atveriant duomenis agregavimą reikėtų naudoti tik tiek, kiek yra būtina užtikrinant konfidencialumą. Tam yra taikomi [konkretūs metodai](https://data.gov.lt/opening/learningmaterial/11).

Žinoma, šis komentaras yra tik patariamojo pobūdžio. Šią vietą būtų privaloma taisyti tik tuo atveju, jei atsirastu poreikis iš duomenų naudotojų.

Ar kažkas bus keičiama dėl agregavimo, ar paliksite taip, kaip yra?

Reply:
Jeigu netinka kaip yra, galim išvis pašalinti

Reply:
Iš techninės perspektyvos tinka. Iš duomenų kokybės pusės, matyt galima padaryti geriau. Bet šis sprendimas yra jūsų pusėje.